qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -163,15 +163,14 @@
                     take_regular_screendumps = no
                     # Because screendump uses '/dev/shm' by default, it is shared with memory-backend-file.
                     # And this case only supports Linux, and don't need to check screendump. So turn off this feature.
-                    xfstest_pkg = xfstests.tar.bz2
                     filesystems = fs1 fs2
                     fs_source_dir_fs1 = /tmp/virtio_fs1_test
                     fs_source_dir_fs2 = /tmp/virtio_fs2_test
                     fs_target_fs1 = myfs1
                     fs_target_fs2 = myfs2
                     fs_dest_fs1 = '/mnt/${fs_target_fs1}'
                     fs_dest_fs2 = '/mnt/${fs_target_fs2}'
-                    cmd_unpack_xfstest = 'tar -jxvf ${fs_dest_fs1}/${xfstest_pkg} -C /home'
+                    cmd_download_xfstest = 'cd /home && git clone https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git'
```

Comment:
How about below?
git clone https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git /home

Reply:
hit below issue:
fatal: destination path '/home' already exists and is not an empty directory.

Reply:
yes, it need to make a empty directory. it's ok for me for your code.Running it from x86 part now.

Reply:
There is a question:
```
[root@hp-dl388g8-16 ~]# cd /home && git clone https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git
Cloning into 'xfstests-dev'...
remote: Enumerating objects: 235, done.
remote: Counting objects: 100% (235/235), done.
remote: Compressing objects: 100% (114/114), done.
remote: Total 36798 (delta 146), reused 187 (delta 119), pack-reused 36563
Receiving objects: 100% (36798/36798), 8.55 MiB | 768.00 KiB/s, done.
Resolving deltas: 100% (28462/28462), done.
[root@hp-dl388g8-16 home]# cd /home && git clone https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git
fatal: destination path 'xfstests-dev' already exists and is not an empty directory.
```
From the above testing, found that could not git clone if the xfstests-dev directory is not deleted.
So I suggest we could clean it before downloading.
e.g:
`cmd_download_xfstest = 'cd /home && rm -rf xfstests-dev && git clone https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git'`

Reply:
To be honest, xfstest with `image_snapshot = yes` 
So every test will be a new env, I am not sure if we will hit the above issue.
But this addition is more rigorous, so I will update it.